### DISCUSSION TITLE
A unique verification for NFT artists?

### DISCUSSION THREAD

**Discussion by ShhubhamA** [deso-protocol/dips#103] [Q&A] [answered]
I think we need a verification system for NFT artists 
So that we know which artists are original and the NFT we are buying is not A fake copied screenshotted shit.

i guess Low mint costs are great to increase the volume of nft sales  for sure but also increased NFT spam and fake are spurting more than ever

**Comment by bloated-devish** [comment, marked as answer]
Should this be something that the community (as in 3rd party Bitclout Developers) should build? Bitclout.com node can also integrate this (just like they are today doing for top projects and top diamonds from bithunt and bitcloutpulse).
